### PR TITLE
feat: added results page

### PR DESCRIPTION
**Pull request by aliamaanansari**

### CODE REVIEW

File: result-page/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/result-page/styles.css" />
+    <link rel="shortcut icon" href="assets/favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="/assets/colors.css" />
+
+    <title>Quiz App</title>
```
Comment (kjais1720) [inline result-page/index.html:11]: Write the page name here as well

File: result-page/styles.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: #0e131f;
+  color: #fff;
```
Comment (kjais1720) [inline result-page/styles.css:5]: use variables for colors, try not to use hard coded values for colors anywhere

File: result-page/styles.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+}
+.option-4 {
+  display: flex;
+  justify-content: center;
+  padding: 1rem;
+  margin: 2rem 1rem 2rem 2rem;
+  background-color: green;
+  padding: 1rem 1rem 1rem 1rem;
+  border-radius: 1rem;
+}
```
Comment (kjais1720) [inline result-page/styles.css:70]: all the options have the same properties, you can give them all a class like `.option` and then apply these styles to the `.option` class

Comment (kjais1720) [reply result-page/styles.css:70]: then you can change the color of any option targeting that option's class (`.option- 1| 2 | 3 | 4`)

**Review by kjais1720 — CHANGES_REQUESTED** [review]

File: result-page/styles.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+.final-results {
+  display: flex;
+  justify-content: center;
+  font-size: xx-large;
```
Comment (Sameony) [inline result-page/styles.css:74]: Several classes could've been avoided with the use od proper utility classes, do check them out after completing the project

**Review by Sameony — APPROVED** [review]
Do follow up on the comments suggested by kritu

**Pull request merged**